internal/api/system.go

```diff
@@ -4,25 +4,54 @@ import (
 	log "github.com/ChainSafe/log15"
 )
 
-type systemModule struct {
-	p2p     P2pApi
-	runtime RuntimeApi
+type p2pModule struct {
```

Comment:
Please move these to seperate files, lets create a new package `internal/api/modules` for them